### PR TITLE
Release notes for bug fix

### PR DESCRIPTION
**Pull request by Eirik0**
These are release notes for https://github.com/zcash/zcash/pull/4177

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -24,6 +24,16 @@ minutes, but this can be configured with the parameter
 
 For full details see ZIP 401.
 
+Async-Operations Incorrectly Reporting Success
```
Comment (daira) [inline doc/release-notes.md]: I suggest "Asynchronous Operations" (and similarly below). Does not block.

File: doc/release-notes.md [hunk 2]
```diff
@@ -24,6 +24,16 @@ minutes, but this can be configured with the parameter
 
 For full details see ZIP 401.
 
+Asynchronous Operations Incorrectly Reporting Success
+-----------------------------------------------------
+We fixed an issue where asynchronous operations were sometimes reporting sucess
+when they had actually failed. One way this could occur when trying using
```
Comment (daira) [inline doc/release-notes.md]:
```suggestion
when they had actually failed. One way this could occur was when trying to use
```

**Comment by Eirik0** [comment]
@ebfull says: "ACK. can’t add ack yet (on phone) but you can proceed by noting that I ack’d in a comment"

**Comment by Eirik0** [comment]
@zkbot r+

**Pull request merged**